detox/runners/jest/adapter.js

```diff
@@ -1,4 +1,4 @@
 const detox = require('../../src/index');
-const DetoxJestAdapter = require('./DetoxJestAdapter');
+const DetoxLifecycleAdapter = require('./DetoxLifecycleAdapter');
 
-module.exports = new DetoxJestAdapter(detox);
\ No newline at end of file
+module.exports = new DetoxLifecycleAdapter(detox);
```

Comment:
What if we write a small class that composes both LifecycleAdapter and TraceReporter into a single adapter? That would eliminate any need to change anything in `e2e/init.js` files from our users' side.

Reply:
Of course, there should be some convenient static method to enable/disable trace reporter, when requiring this facade.

Reply:
I take my words back. After spending some time with the code, I think that TraceReporter should be a recommended third-party npm module to use with Detox + Jest. In the next PR, we can covert it in docs and encourage using it.